src/lancie-admin-progress/lancie-admin-progress.html

```diff
@@ -78,17 +77,16 @@
       });
 
       for (var c in this.consumptions) {
-        this.addProgressMeter(c, this.consumptions[c], this.tickets);
+        this.addProgressMeter(c, this.consumptions[c]);
       }
     }
   }
 
   // Adds a new progressMeter with the Polymer push function to notify
-  addProgressMeter(name, count, total) {
+  addProgressMeter(name, count) {
```

Comment:
I think that its also useful to check whether a progress meter with the same name doesn't exist already? And remove it if it does exist. Also, I think that it would be even better if the logic for the progress status was done in the API and that one request was sufficient, but that would be for another PR

Reply:
I agree for the api part, it would be better to track it there. I'll add the check though.